LoRaEngine/modules/LoRaWanPktFwdModule/Dockerfile

```diff
@@ -6,16 +6,15 @@ RUN apt-get install -y --no-install-recommends apt-utils build-essential
 RUN git clone https://github.com/Lora-net/packet_forwarder.git
 RUN git clone https://github.com/Lora-net/lora_gateway.git
 RUN sed -i "s|/dev/spidev0.0|/dev/spidev2.0|g" ./lora_gateway/libloragw/src/loragw_spi.native.c
-RUN sed -i "s|IOT_SK_SX1301_RESET_PIN=7|IOT_SK_SX1301_RESET_PIN=5|g" ./lora_gateway/reset_lgw.sh
 WORKDIR /packet_forwarder
 RUN ./compile.sh
 WORKDIR /build
 
-FROM microsoft/dotnet:2.0-runtime AS exec
+FROM debian:stretch-slim AS exec
```

Comment:
Not sure about this but Based on https://hub.docker.com/r/microsoft/dotnet/~/dockerfile/ it looks like the dotnet is currently based on Jessie. If we want to do image sharing we might change to Jessie rather than Stretch? (TBD)

Reply:
I agree either we stay with the dotnet one or better we test and use the Jessie that dotnet uses. Saving space and download time.

Reply:
Thats why I switched away from .NET. Stretch is the one that is below the 2.0 .NET image we used. I can upgrade if you want, no problem.

Reply:
I think its good to have the same based linux image for both modules so I would stay with debian:stretch-slim here.